1_fetch/unzip_file.py

```diff
@@ -0,0 +1,16 @@
+import zipfile
+import os
+
+def unzip_file(file_path, out_dir, sb_file):
+    file_path = os.path.join(out_dir, sb_file)
```

Comment:
`file_path` is an argument to `unzip_file`, but its value is immediately reassigned in the first line of the function? Either remove `file_path` as an argument or remove this line and remove `sb_file` as an argument.